telemetry/telemetry.go

```diff
@@ -133,12 +133,7 @@ func getDistinctID() string {
 		frontClient := rest.NewFrontClient(rest.Domain, GetClientID())
 		userResponse, err := frontClient.GetUser(ctx)
 		if err == nil {
-			if userResponse.Email != "" {
-				return userResponse.Email
-			}
-
-			// Use the user ID if no email is present;
-			// this should be fixed in the current backend.
+			// User postman user's userID as distinctID for telemetry
```

Comment:
Please fix the commenst above (line 116, 129) with what the current strategy is.

Reply:
Looks like this is obsoleted by #262?